xmodule/tests/test_library_content.py

```diff
@@ -791,48 +791,3 @@ def test_author_view(self):
         assert '<li>html 2</li>' in rendered.content
         assert '<li>html 3</li>' in rendered.content
         assert '<li>html 4</li>' in rendered.content
-
-    def test_xml_export_import_cycle(self):
```

Comment:
There's another test_xml_export_import_cycle test method in this module, which sufficiently tests the OLX round trip.

The extra assertions that this test has, in particular `child.xml_attributes.get('upstream') is not None`, do not actually test anything worthwhile.  The value of `upstream` on these children is `""`, which is not None, but also, it may as well be None.

Seeing that this test method is part of `TestLibraryContentRender`, I believe that this test method was added back when the _rendering of the legacy library content block would trigger the migration_. This is no longer the case, but the test never failed when we switched to user-triggered migration(oops). Anyway, it's obsolete, so we remove it.